data/virtualization/autoyast/guest_12.xml.ep

```diff
@@ -0,0 +1,334 @@
+      </listentry>
+      % for my $repo (@$repos) {
+      <listentry>
+        <media_url><%= $repo %></media_url>
+      </listentry>
+      % }
+    </add_on_products>
+  </add-on>
+  <firewall>
+    <enable_firewall config:type="boolean">false</enable_firewall>
+    <start_firewall config:type="boolean">false</start_firewall>
+  </firewall>
+  <keyboard>
+    <keymap>english-us</keymap>
+  </keyboard>
+  <language>
+    <language>en_US</language>
+    <languages/>
+  </language>
+  <login_settings/>
+  <networking>
+    <dhcp_options>
+    <dhclient_client_id><%= $vm_name %></dhclient_client_id>
+    <dhclient_hostname_option>AUTO</dhclient_hostname_option>
+    </dhcp_options>
+    <dns>
+    <dhcp_hostname config:type="boolean">false</dhcp_hostname>
+    <hostname><%= $vm_name %></hostname>
+    <resolv_conf_policy>auto</resolv_conf_policy>
+    <write_hostname config:type="boolean">true</write_hostname>
+    <nameservers config:type="list">
+    <nameserver>192.168.122.1</nameserver>
+    </nameservers>
+    </dns>
+    <interfaces config:type="list">
+      <interface>
+      <bootproto>static</bootproto>
+      <device>eth0</device>
+      <ipaddr>{{IP}}</ipaddr>
+      <netmask>255.255.255.0</netmask>
+      <network>192.168.122.0</network>
+      <prefixlen>24</prefixlen>
+      <startmode>auto</startmode>
+      </interface>
+    </interfaces>
+    <ipv6 config:type="boolean">true</ipv6>
+    <keep_install_network config:type="boolean">true</keep_install_network>
+    <managed config:type="boolean">false</managed>
+    <net-udev config:type="list">
+      <rule>
+       <name>eth0</name>
+       <rule>ATTR{address}</rule>
+       <value>{{MAC}}</value>
+      </rule>
+    </net-udev>
+    <routing>
+    <ipv4_forward config:type="boolean">false</ipv4_forward>
+    <ipv6_forward config:type="boolean">false</ipv6_forward>
+    <routes config:type="list">
+    <route>
+    <destination>default</destination>
+    <device>-</device>
+    <gateway>192.168.122.1</gateway>
+    <netmask>-</netmask>
+    </route>
+    </routes>
+    </routing>
+  </networking>
+  <kdump>
+    % if ($vm_name !~ /PV/) {
+    <add_crash_kernel config:type="boolean">true</add_crash_kernel>
+    <crash_kernel config:type="list">
+      <listentry>168M</listentry>
+    </crash_kernel>
+    % } else {
+    <add_crash_kernel config:type="boolean">false</add_crash_kernel>
+    % }
+  </kdump>
+  <partitioning config:type="list">
+    <drive>
+    % if ($vm_name =~ /PV|HVM/) {
+      <device>/dev/xvda</device>
+    % } else {
+      <device>/dev/vda</device>
+    % }
+      <disklabel>msdos</disklabel>
+      <enable_snapshots config:type="boolean">false</enable_snapshots>
+      <initialize config:type="boolean">true</initialize>
+      <partitions config:type="list">
+        <partition>
+          <create config:type="boolean">true</create>
+          <crypt_fs config:type="boolean">false</crypt_fs>
+          <filesystem config:type="symbol">btrfs</filesystem>
```

Comment:
I had a check in existing autoyast files under data/autoyast_[kvm|xen], for 12sp3->15sp3, they have already all used btrfs, so using btrfs globally is fine. 
However, subvolumes have all been used in them. This PR does not list subvolumes, at least for libvirt/images. Do you check what finally turns out? Subvolumes created under the device mounted at /? Besides, uefi vm is on the way to be added. A UEFI machine requires an EFI system partition that must be mounted to /boot/efi. This partition must be formatted with the FAT32 file system. Please take into consideration for easy extension for that.
Finally how about snapshots? No explicit setting here. If different systems have different default setting on this, some enable some disable, will 16G be large enough with space buffer for that?